spec/engine_spec.rb

```diff
@@ -229,6 +230,33 @@
           .to eq SplitIoClient::Engine::Models::Treatment::CONTROL
         expect(log.string).to include 'get_treatment: attributes must be of type Hash'
       end
+
+      it 'returns CONTROL and logs warning when ready and split does not exist' do
+        # allow(subject).to receive(:ready?).and_return(true)
```

Comment:
is this commented for any reason?

Reply:
no reason other than forgot to remove them. Removed.